src/command_args.rs

```diff
@@ -12,35 +12,47 @@ use serde::Deserialize;
 use std::error::Error;
 
 pub const OPTION_NAME_CONFIG: &str = "config";
+
+/// # Signer Config
 pub const OPTION_NAME_TO_ADDRESS: &str = "coinbase_pay_to_address";
 pub const OPTION_NAME_PUBLIC_KEY: &str = "publickeys";
 pub const OPTION_NAME_PRIVATE_KEY: &str = "privatekey";
 pub const OPTION_NAME_THRESHOLD: &str = "threshold";
 pub const OPTION_NAME_MASTER_FLAG: &str = "master_flag";
+
+/// # RPC Config
 pub const OPTION_NAME_RPC_ENDPOINT_HOST: &str = "rpc_endpoint_host";
 pub const OPTION_NAME_RPC_ENDPOINT_PORT: &str = "rpc_endpoint_port";
 pub const OPTION_NAME_RPC_ENDPOINT_USER: &str = "rpc_endpoint_user";
 pub const OPTION_NAME_RPC_ENDPOINT_PASS: &str = "rpc_endpoint_pass";
 
+/// # Redis Config
 pub const OPTION_NAME_REDIS_HOST: &str = "redis_host";
 pub const OPTION_NAME_REDIS_PORT: &str = "redis_port";
 
+/// # General Config
 /// round category params.
 pub const OPTION_NAME_ROUND_DURATION: &str = "round_duration";
-
 /// log category params.
 pub const OPTION_NAME_LOG_QUIET: &str = "log_quiet";
 pub const OPTION_NAME_LOG_LEVEL: &str = "log_level";
-
+/// daemonize
+pub const OPTION_NAME_DAEMON: &str = "daemon";
+pub const OPTION_NAME_PID: &str = "pid";
+pub const OPTION_NAME_LOG_FILE: &str = "log_file";
+/// Others
 pub const OPTION_NAME_SKIP_WAITING_IBD: &str = "skip_waiting_ibd";
 
+/// # Default Values
 pub const DEFAULT_RPC_HOST: &str = "127.0.0.1";
 pub const DEFAULT_RPC_PORT: &str = "2377";
 pub const DEFAULT_RPC_USERNAME: &str = "";
 pub const DEFAULT_RPC_PASSWORD: &str = "";
 pub const DEFAULT_REDIS_HOST: &str = "127.0.0.1";
 pub const DEFAULT_REDIS_PORT: &str = "6379";
 pub const DEFAULT_LOG_LEVEL: &str = "info";
+pub const DEFAULT_PID: &str = "/tmp/tapyrus-signer.pid";
```

Comment:
The tmp directory path is OS dependent. It is better to get it with `std::env::temp_dir`.  `/var/log` too.

Reply:
Ok, I will fix it.

Reply:
Regarding log file, which place is better? I have no idea. so i'm thinking that i'm going to remove default value.

Reply:
How about same `tmp` dir, instead of remove?

Reply:
It's ok too. I will.